docs/departments/law-enforcement/standard-operating-procedures.mdx

```diff
@@ -112,22 +114,29 @@ Assistant Commissioner
 
 Deputy Commissioner
 
-Department Supervisors
+Department Supervisors (With the option of Feild trainer (FTO))
```

Comment:
Should capitalize "Trainer" and add the word "Officer" to complete the FTO abbreviation